packages/contracts/src/diamond/facets/ManagerFacet.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.3;
+
+import {
+    AppStorage,
+    Modifiers,
+    UBQ_MINTER_ROLE,
+    PAUSER_ROLE,
+    COUPON_MANAGER_ROLE,
+    BONDING_MANAGER_ROLE,
+    INCENTIVE_MANAGER_ROLE,
+    UBQ_TOKEN_MANAGER_ROLE    
+} from "../libraries/LibAppStorage.sol";
+import { AccessControlStorage } from "solidstate/contracts/access/access_control/AccessControlStorage.sol";
+
+contract ManagerFacet is Modifiers {
```

Comment:
This facet is still missing a couple of functions from the original UbiquityAlgorithmicDollarManager.sol. its also same in unit tests, we should have all of them in both implementation and tests.

Reply:
> This facet is still missing a couple of functions from the original UbiquityAlgorithmicDollarManager.sol. its also same in unit tests, we should have all of them in both implementation and tests.

Yeah, that was intended for a quick review. I will add remained functions

Reply:
Please make sure it contains everything when you request for review.